Dockerfile

```diff
@@ -12,20 +12,24 @@ RUN apt install bluez -y
 
 RUN apt install build-essential libglib2.0-dev libical-dev libreadline-dev libudev-dev libdbus-1-dev libdbus-glib-1-dev bluetooth libbluetooth-dev usbutils -y
 
-RUN pip3 install idasen-controller==2.0.1
+RUN export SKIP_CYTHON=false
```

Comment:
Can you please add a comment explaining why this is needed?

Reply:
Was needed regarding this [issue with dbus-fast](https://github.com/Bluetooth-Devices/dbus-fast/issues/237#issuecomment-1705993762).